.size-limit.cjs

```diff
@@ -7,7 +7,7 @@ function modifyWebpackConfig(config) {
 module.exports = [
   {
     path: 'dist/chart.js',
-    limit: '75 KB',
+    limit: '77 KB',
```

Comment:
this is quite the size increase. if we add 2KB for every file that's going to be quite substantial. what's lead to the increase?

Reply:
<img width="1334" alt="Снимок экрана 2022-08-13 в 01 11 04" src="https://user-images.githubusercontent.com/986756/184445482-7881c9bf-9d4c-4ce2-ac40-002b252216c5.png">

Just a little bit different output because of swc. size-limit doesn't apply minification, only compression

Reply:
Ok, really just spacing so far then, which won't matter after minification